pdostal change maintainer

pdostal opened the pull request:
- Transfer ownership of virtualization tests from me, @grisu48 and @baierjan to virtualization team
- Transfer ownership of NFS4, Apache2 and Salt from me to QE Core
- Transfer ownership of Public Cloud tests to qe-c team
- Get rid of ownership of remote_desktop/screensharing_available test
    I unfortunately don't know who to transfer it to.
- I stay maintainer for Ansible, Wireguard, openVPN, SSHD, SLP, SSHd, RPM, NginX

pdostal committed:
Transfer ownership of virtualization tests from me to virtualization team

pdostal committed:
Transfer ownership of NFS4, Apache2 and Salt from me to QE Core

pdostal committed:
Transfer ownership of Public Cloud tests to qe-c team

grisu48 reviewed (approved):
LGTM, but there are CI issues.

pdostal committed:
Get rid of ownership of tests/x11/remote_desktop/screensharing_available.pm
This test is out of my scope. I unfortunately don't know who to transfer it to.

alice-suse reviewed (approved):
LGTM

foursixnine commented on tests/x11/remote_desktop/screensharing_available.pm:
```diff
@@ -12,7 +12,7 @@
 #   - If wayland is detected, record_soft_failure (boo#1137569)
 # - Finish with ctrl-q
-# Maintainer: Pavel Dostál <pdostal@suse.cz>
+# Maintainer:
```
this would be Desktop team, but qe-core can be it for now

foursixnine replied:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/actions/runs/4499546432/jobs/7917547747?pr=16691#step:6:83

pdostal replied:
Do you have their e-mail and some github nickname to contact for approval?

pdostal replied:
It is QE Core for now. You may merge.

foursixnine commented:
> LGTM, but there are CI issues.

Captured it: https://progress.opensuse.org/issues/126518
